src/Include/Functions.php

```diff
@@ -1096,8 +1096,8 @@ function validateCustomField($type, &$data, $col_Name, ?array &$aErrors): bool
                 if (!is_numeric($data) || strlen($data) != 4) {
```

Comment:
Let's remove the strlen check here
```suggestion
                if (!is_numeric($data)) {
```